src/foam/core/Boot.js

```diff
@@ -192,28 +192,49 @@ foam.LIB({
       /* Creates a Foam class from a plain-old-object definition.
           @method CLASS
           @memberof module:foam */
-      foam.CLASS = function(m) {
-        var cls   = m.class ? foam.lookup(m.class) : foam.core.Model;
-        var model = cls.create(m);
-        model.validate();
-        var cls = model.buildClass();
+      foam.CLASS = function(clsDef) {
```

Comment:
That's true. I renamed things b/c I felt it clarified things, and is consistent with proposed changes in #263. I'll definitely change it back of the other proposed changes get dropped.